src/entities/Bullet_test.js

```diff
@@ -52,3 +53,48 @@ describe('Bullet', () => {
     });
   });
 });
+
+describe('PlasmaBullet', () => {
+  const position = new Vec2(2, 2);
+  const direction = new Vec2(0, 1);
+
+  it('is orange', () => {
+    const bullet = new PlasmaBullet(position, direction, null);
+    assert(bullet.color == 'orange');
+  });
+
+  describe('´Temperature', () => {
```

Comment:
`Temperature` isn't a class (upper case first letter). It's not even a property name. Either remove it, or rename it `heat`.